win32/installer/hexchat.iss.tt

```diff
@@ -309,6 +323,9 @@ begin
 #endif
 		DOTNET := 'https://dl.hexchat.net/misc/dotnet_40.exe';
 		SPELL := 'https://dl.hexchat.net/hexchat/HexChat%20Spelling%20Dictionaries%20r2.exe';
+		SYSPIN := 'https://www.technosys.net/download.aspx?file=syspin.exe';
```

Comment:
I don't want to add a third party executable to our installer unless I really trust it. (Which I do for python.org and microsoft).

Reply:
Can't argue here.
The only issue is finding another way to do this. From what I have found so far, microsoft removed the easy way to do this programmatically...
I'll try to find another way.
However, if you want, it's possible to merge everything except the pinning to taskbar and try to work on it separately.

Reply:
Yes lets keep that feature for another day. Its not super valuable IMO.

Reply:
Removed.